src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -111,5 +111,15 @@ public new JwtBearerEvents Events
         /// from returning an error and an error_description in the WWW-Authenticate header.
         /// </summary>
         public bool IncludeErrorDetails { get; set; } = true;
+
+        /// <summary>
+        /// 1 day is the default time interval that afterwards, <see cref="ConfigurationManager" /> will obtain new configuration.
+        /// </summary>
+        public TimeSpan AutomaticRefreshInterval { get; set; } = ConfigurationManager<OpenIdConnectConfiguration>.DefaultAutomaticRefreshInterval;
+
+        /// <summary>
+        /// 30 seconds is the default, minimum time interval that must pass for <see cref="ConfigurationManager" /> to obtain a new configuration.
```

Comment:
The [doc comments for the ConfigurationManager](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/06cae9a6d422dac7b76f1db9de032b98c1d8c836/src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs#L131-L159) make it clearer why AutomaticRefreshInterval and RefreshInterval exist as two different properties. I would just change these doc comments to match. E.g.

```suggestion
        /// The minimum time between <see cref="ConfigurationManager" /> retrievals, in the event that a retrieval failed, or that a refresh was explicitly requested. 30 seconds is the default.
```

I don't care about the exact wording, but including "in the event that a retrieval failed, or that a refresh was explicitly requested" makes it a lot clearer to me.